### ISSUE TITLE
psalm disable PossiblyUndefinedArrayOffset 

### ISSUE THREAD

**Issue by Traydr** [Traydr/ppm#4]
Attempting to do this, because psalm throws an error whenever:

```php
<?php include_once($_SERVER['DOCUMENT_ROOT'] . "/generic/header.php") ?>
```

shows up in code.

See: https://github.com/Traydr/ppm/actions/runs/4788705369/jobs/8515660809#step:5:362

**Comment by Traydr** [comment]
Honestly, getting trouble to run `psalter` as well, seems to not be applying fixes very well.

**Comment by Traydr** [comment]
fixed in commit: 78507b4
